dev-libs/bglibs/bglibs-1.106-r2.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
Thanks @mm1ke , this package sorely needed it. Instead of bumping the current version, could you try adding the latest upstream version instead?

Reply:
Hmm, i can do it on my todo list. I saw there is a newer version of that package, but to be honest i don't use any of it's customers so i didn't bother looking further into it..